nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -581,7 +581,21 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - The `miller` package has been upgraded from 5.10.3 to [6.2.0](https://github.com/johnkerl/miller/releases/tag/v6.2.0). See [What's new in Miller 6](https://miller.readthedocs.io/en/latest/new-in-miller-6).
 
-- MultiMC has been replaced with the fork PolyMC due to upstream developers being hostile to 3rd party package maintainers. PolyMC removes all MultiMC branding and is aimed at providing proper 3rd party packages like the one contained in Nixpkgs. This change affects the data folder where game instances and other save and configuration files are stored. Users with existing installations should rename `~/.local/share/multimc` to `~/.local/share/polymc`. The main config file's path has also moved from `~/.local/share/multimc/multimc.cfg` to `~/.local/share/polymc/polymc.cfg`.
+- MultiMC has been replaced with the fork PrismLauncher due to upstream
+  developers being hostile to 3rd party package maintainers. PrismLauncher
+  removes all MultiMC branding and is aimed at providing proper 3rd party
+  packages like the one contained in Nixpkgs. This change affects the data
+  folder where game instances and other save and configuration files are stored.
+  Users with existing installations should rename `~/.local/share/multimc` to
+  `~/.local/share/PolyMC`. The main config file's path has also moved from
+  `~/.local/share/multimc/multimc.cfg` to
+  `~/.local/share/PolyMC/prismlauncher.cfg`.
```

Comment:
```suggestion
  Users with existing installations should rename `~/.local/share/multimc` to
  `~/.local/share/PrismLauncher`. The main config file's path has also moved from
  `~/.local/share/multimc/multimc.cfg` to
  `~/.local/share/PrismLauncher/prismlauncher.cfg`.
```

Reply:
On my machine this isn't the case (i.e. there is no .local/share/PrismLauncher folder and .local/share/PolyMC/prismlauncher.cfg exists). Can you confirm that the commit this was changed in is definitely after the one we're using here?

Reply:
The launcher looks for the old directories first. This is our current way of "migrating" users. In the future `.local/share/PrismLauncher` should be preferred data directory.

Reply:
> The launcher looks for the old directories first. This is our current way of "migrating" users. In the future `.local/share/PrismLauncher` should be preferred data directory.

Gotcha, in that case I've updated the release notes to reflect this